raiden_installer/token_exchange.py

```diff
@@ -279,10 +284,11 @@ def _calculate_transaction_costs(self, token_amount: TokenAmount, account: Accou
             "exchange_rate": exchange_rate,
         }
 
-    def buy_tokens(self, account: Account, token_amount: TokenAmount, transaction_costs=dict()):
-        if transaction_costs:
+    def buy_tokens(self, account: Account, token_amount: TokenAmount, transaction_costs=None):
```

Comment:
Just out of curiousity. Whats the reason for that change?

Reply:
Mutable default arguments are dangerous: https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/